client/my-sites/plans-features-main/components/plans-page-subheader.tsx

```diff
@@ -147,9 +150,11 @@ const PlansPageSubheader = ( {
 		return getCreateWithBigSky();
 	}, [] );
 
-	return (
-		<>
-			{ createWithBigSky && (
+	const isOnboarding = isOnboardingFlow( flowName ?? null );
+
+	const renderSubheader = () => {
+		if ( createWithBigSky ) {
```

Comment:
I think there's a subtle logic change here. I think if `createWithBigSky` is true then this function should return 

```ts
<>
	<Subheader>
		{ translate( 'Build your ....' ) }
	</Subheader>
	{ showPlanBenefits && <PlanBenefitHeader />
</>
```

I don't think I can test this logic anymore though, because you can't choose big sky before getting to the plans step anymore. The plans component in Calypso doesn't show this sub header

Reply:
You are absolutely correct. The big sky header can be rendered along with the plans benefit header. It seems like we should remove the big sky related copy here since they are basically unused now that BigSky is after plans page?